test/e2erp/client_test.go

```diff
@@ -25,19 +27,24 @@ type AzureConfig struct {
 	ClientID        string   `envconfig:"AZURE_CLIENT_ID" required:"true"`
 	ClientSecret    string   `envconfig:"AZURE_CLIENT_SECRET" required:"true"`
 	ResourceGroup   string   `envconfig:"RESOURCEGROUP" required:"true"`
+	ClusterName     string   `envconfig:"CLUSTERNAME"`
```

Comment:
Let's add this when we are going to use it. Remove it for now.

Reply:
@kargakis actually I use it right now, the `az openshift` tool allows you to create several cluster per "parent" resourcegroup and of course the cluster and resource group may not match, so I feel like we should account for that already

Reply:
I meant are there any plans to use this in CI?

Reply:
Ah, you're right. Gonna remove.